quantum/quantum_keycodes.h

```diff
@@ -453,8 +453,8 @@ enum quantum_keycodes {
 
     UNICODE_MODE_OSX,
     UNICODE_MODE_LNX,
-    UNICODE_MODE_BSD,
     UNICODE_MODE_WIN,
+    UNICODE_MODE_BSD,
```

Comment:
This is so that it's consistent with the order of declaration in `process_unicode_common.h`. To be honest, I'd prefer it if BSD would come before WIN everywhere, so that everything is logically grouped, but we can do that in a followup PR.